packages/editor-ui/src/components/mixins/deviceSupportHelpers.ts

```diff
@@ -3,7 +3,8 @@ import Vue from 'vue';
 export const deviceSupportHelpers = Vue.extend({
 	data() {
 		return {
-			isTouchDevice: 'ontouchstart' in window || navigator.maxTouchPoints,
+			// @ts-ignore msMaxTouchPoints is deprecated but must fix tablet bugs before fixing this.. otherwise breaks touchscreen computers
+			isTouchDevice: 'ontouchstart' in window || navigator.msMaxTouchPoints,
```

Comment:
Oddly, `msMaxTouchPoints` and `platform` are both deprecated but `msMaxTouchPoints` returns `undefined` and `platform` still returns a string. Wondering if we might have an issue once `platform` starts returning `undefined`.

Reply:
yes would have to handle platform separately as tech debt